Server/server.js

```diff
@@ -642,8 +642,23 @@ app.post('/api/get-MentorData', function(req, res, next) {
     }
   })
 
+  app.get('/api/quizsearch',async(req,res)=>{
+    const quizzes = await Quiz.find({});
+    //console.log(quizzes)
+    if(!quizzes){
+      console.log("no quiz was found")
+      return res.sendStatus(404)
+    } else {
+      return res.status(200).send(quizzes);
+    }
+    /*mongoose.connection.db.collection('quizzes').find({authorID: req.params.aoid}).toArray().then(collection => {  
+      //console.log("check quiz: " + collection);
+      res.status(200).json({collection})
+    }); */
```

Comment:
are we free to remove lines 654-657 here? It looks like this code is listed elsewhere so we won't lost it forever